lessons/beginners/class/index.md

```diff
@@ -320,60 +340,18 @@ class Kotatko:
         self.jmeno = jmeno
 
     def __str__(self):
-        return '<Kotatko jmenem {}>'.format(self.jmeno)
+        return f'<Kotatko jmenem {self.jmeno}>'
 
     def zamnoukej(self):
-        print("{}: Mňau!".format(self.jmeno))
+        print(f"{self.jmeno}: Mňau!")
 
     def snez(self, jidlo):
-        print("{}: Mňau mňau! {} mi chutná!".format(self.jmeno, jidlo))
+        print(f"{self.jmeno}: Mňau mňau! {jidlo} mi chutná!")
 
 mourek = Kotatko('Mourek')
 print(mourek)
 ```
 
-## Cvičení: Kočka
-
-Teď, když už umíš dělat koťátka, zkus vytvořit třídu pro kočku.
-
-- Kočka umí mňoukat metodou `zamnoukej`.
-- Kočka má na začátku (při vytvoření) 9 životů
-(nemůže mít nikdy víc než 9 nebo míň než 0!).
-- Kočka umí říct, jestli je živá (nemá 0 životů), metodou `je_ziva`.
-- Kočka může ztratit život metodou `uber_zivot`.
-- Kočku můžeš nakrmit metodou `snez`, která bere 1 argument -
-nějaké konkrétní jídlo (řetězec). Pokud je toto jídlo `"ryba"`, kočce se obnoví
-jeden život (pokud teda už není mrtvá, nebo nemá maximální počet životů).
-
-{% filter solution %}
-```python
-class Kocka:
-    def __init__(self):         # Init funkce nemusi brat jako parametr
-        self.pocet_zivotu = 9   # pocet zivotu, ten je pokazde 9.
-
-    def zamnoukej(self):
-        print("Mnau, mnau, mnauuu!")
-
-    def je_ziva(self):
-        return self.pocet_zivotu > 0
-
-    def uber_zivot(self):
-        if not self.je_ziva():
-            print("Nemuzes zabit uz mrtvou kocku!")
-        else:
-            self.pocet_zivotu -= 1
-
-    def snez(self, jidlo):
-        if not self.je_ziva():
-            print("Je zbytecne krmit mrtvou kocku!")
-            return
-        if jidlo == "ryba" and self.pocet_zivotu < 9:
-            self.pocet_zivotu += 1
-            print("Kocka spapala rybu a obnovil se ji jeden zivot.")
-        else:
-            print("Kocka se krmi.")
-```
-{% endfilter %}
```

Comment:
@frenzymadness Zvládnete to bez úkolu v materiálech?

Reply:
Myslím si, že ano. Navrhnu jeho transformaci na domácí projekt. Máme v plánu začít s domácími projekty ještě na hodině, takže to snad nebude žádná velká změna. Cc @adelpopelkova @Disi77